Automattic/wp-calypso
# Gutenberg: Update Publicize to use inline icons

#### Changes proposed in this Pull Request

* Introduce a new `PublicizeServiceIcon` component with inline icons for Publicize.
* Use that component to render the same icons in any environment.

#### Testing instructions

* Checkout this branch in Calypso.
* Load http://calypso.localhost:3000/block-editor/post/:site where `:site` is a WP.com simple site.
* Click the Jetpack icon in the toolbar.
* Connect one of each Publicize services.
* Verify icons look well.
* Start a JN site with this branch - `gutenpack-jn`.
* Connect the site and activate recommended features.
* Start a new post.
* Click the Jetpack icon in the toolbar, or the Publish button.
* Connect one of each Publicize services.
* Verify icons look well.

Fixes #28953
